summoner-cli/src/Summoner/GhcVer.hs

```diff
@@ -87,7 +87,7 @@ baseVerPvp = \case
     Ghc844  -> Pvp 4 11 1 0
     Ghc865  -> Pvp 4 12 0 0
     Ghc884  -> Pvp 4 13 0 0
-    Ghc8103 -> Pvp 4 14 1 0
+    Ghc8107 -> Pvp 4 14 1 0
```

Comment:
They bumped minor version for `base` in minor GHC releases 😒 
```suggestion
    Ghc8107 -> Pvp 4 14 3 0
```

Reply:
What?!